microservice/microserviceTests/microserviceTests.csproj

```diff
@@ -13,6 +13,7 @@
         <OutputType>Library</OutputType>
 
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <RollForward>LatestMajor</RollForward>
```

Comment:
What is RollForward? Is it pertinent to Optional types in JSON serialization?

Reply:
oh yeah, this is a piggy back change :+1: 

It means that if you only have net8 installed (like I do, now), the project _allows_ itself to be run even if it is targeting net6. 

In this case, the unit test project is saying, 
"hey, I am at minimum a net 6 project!"
"but also, its fine to use net7 or 8".